docs/guide/advanced/meta.md

```diff
@@ -58,10 +58,12 @@ router.beforeEach((to, from) => {
 
 ## TypeScript
 
-It is possible to type the meta field by extending the `RouteMeta` interface:
+It is possible to type the meta field by extending the `RouteMeta` interface from `vue-router`:
 
 ```ts
-// typings.d.ts or router.ts
+// Ensure this file is parsed as a module regardless of dependencies.
+export {}
```

Comment:
```suggestion
// It can also be added to a `.d.ts` file, in which case you will need to add an export
// to ensure it is treated as a module
export {}
```